### PR TITLE
add lsdcRemote-related scripts

### PR DESCRIPTION
**Pull request by JunAishima**
* use these instead of local copies that have gotten
   out of sync with current LSDC code and conda enviornment
   requirements

### CODE REVIEW

**Review by RobertSchaffer1 — COMMENTED** [review]
Those files are pretty close to identical, are you sure we want two files and not one with a variable set?

File: bin/lsdcRemote [hunk 1]
```diff
@@ -0,0 +1 @@
+script -a -c lsdcRemote.cmd -f lsdcRemoteLog.txt
```
Comment (RobertSchaffer1) [inline bin/lsdcRemote:1]: I'm not super familiar with the script command, but I *think* this will try to run the "lsdcRemote.cmd" file which is now missing (having been renamed to "lsdcRemote_amx.cmd" and "lsdcRemote_fmx.cmd").

Comment (JunAishima) [reply bin/lsdcRemote:1]: The procedure for creating the symlinks will help explain this. Probably should be documented!!

On both AMX and FMX, the lsdcServer and lsdcRemote commands are symlinked from /usr/local/bin without any changes to filename.

For the scripts lsdcServer_amx/fmx.cmd and lsdcRemote_amx/fmx.cmd, these files are symlinked and renamed to lsdcServer.cmd and lsdcRemote.cmd, as appropriate for the beamline.

This allows the same general script to be used for both/all beamlines. Eventually this should be added to documentation, and implemented in ansible jobs for installing LSDC

**Comment by JunAishima** [comment]
> Those files are pretty close to identical, are you sure we want two files and not one with a variable set?

I have been making these files separate to reduce the number of conditions which increase the complexity of the code somewhat.

With so many other places where branching makes the code extremely complicated and hard to follow, I am happy to keep this code this way.

**Comment by RobertSchaffer1** [comment]
> > Those files are pretty close to identical, are you sure we want two files and not one with a variable set?
> 
> I have been making these files separate to reduce the number of conditions which increase the complexity of the code somewhat.
> 
> With so many other places where branching makes the code extremely complicated and hard to follow, I am happy to keep this code this way.

Okay, works for me.

**Review by RobertSchaffer1 — APPROVED** [review]

**Review by mskinner5278 — APPROVED** [review]
